parachains/runtimes/starters/shell/Cargo.toml

```diff
@@ -74,6 +74,7 @@ std = [
 	"cumulus-primitives-core/std",
 	"cumulus-primitives-utility/std",
 	"parachain-info/std",
+#	"parachains-common/std",
```

Comment:
```suggestion
```

Reply:
Have uncommented as we're not testing the CI changes in this PR any more.